jobs/src/test/scala/dcos/metronome/repository/impl/kv/ZkJobSpecRepositoryTest.scala

```diff
@@ -1,16 +1,19 @@
 package dcos.metronome
 package repository.impl.kv
 
-import dcos.metronome.model.{ JobId, JobSpec }
+import dcos.metronome.model.{JobId, JobSpec}
 import dcos.metronome.utils.test.Mockito
 import mesosphere.util.state.PersistentStoreWithNestedPathsSupport
 import org.scalatest.FunSuite
 import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.{Millis, Seconds, Span}
 
 import concurrent.Future
 
 class ZkJobSpecRepositoryTest extends FunSuite with Mockito with ScalaFutures {
 
+  override implicit def patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(5, Seconds), interval = Span(500, Millis))
```

Comment:
I don't remember this test failing.

Reply:
huh? I don't understand your comment... Did you check the jenkins job link I provided?

Reply:
sry, I totally missed the fact that that's metronome ^.^